workflow-run-stats/action.yml

```diff
@@ -0,0 +1,141 @@
+name: Workflow Run Statistics
+
+description: |
+  Retrieves workflow run statistics for specified workflows and branches in a repository.
+  Returns counts of successful, failed, and other workflow run statuses within a specified time range.
+
+inputs:
+  github-token:
+    description: GitHub token with access to workflow runs
```

Comment:
suggestions: it may make sense to name the exact permission needed: e.g. actions:readonly.

Reply:
Done.